src/main/webapp/i18n/en/competency.json

```diff
@@ -152,15 +152,27 @@
             "learningPathButton": "Learning Paths",
             "manageLearningPaths": {
                 "title": "Learning Path Management",
+                "health": {
+                    "missing": {
+                        "title": "Missing Learning Paths",
+                        "body": "Some student's have not generated their Learning Paths yet. This is not critical. Their Learning Paths will be created once they request their learning path for the first time.",
+                        "action": "Generate",
+                        "hint": "Generate missing Learning Paths"
+                    }
+                },
                 "isDisabled": "Learning Paths are currently disabled for this course.",
                 "enable": "Enable Learning Paths",
                 "enableHint": "The creation of Learning Paths for every student may take a few minutes.",
                 "search": "Search for Learning Path:",
                 "table": {
                     "name": "Name",
                     "login": "Login",
-                    "progress": "Progress",
-                    "view": "View"
+                    "progress": "Progress"
+                },
+                "progressNav": {
+                    "header": "Learning Path",
+                    "refresh": "Refresh",
+                    "center": "Center view"
```

Comment:
What is a center view supposed to be?

Reply:
The Graph of the learning path is right below this button. Users can zoom in and move around using their mouse and scroll wheel. If they want to zoom out and center the view to see the entire graph again, they can use this button.
I'm open for any suggestions to rename this hint 👍

Reply:
I think this In the context of the ui this is fine 👍